src/pages/components/toggle/usage.mdx

```diff
@@ -52,6 +53,16 @@ Toggle is a control that is used to quickly switch between two possible states.
 Toggles are only used for these binary actions that occur immediately after the
 user “flips the switch”. They are commonly used for “on/off” switches.
 
+<Row>
+<Column colLg={8}>
+
+![Toggle being used in context of a product UI](images/toggle_usage_overview.png)
```

Comment:
**Overview image**
- For this image, you just need to put some gray 30 rules around it on the left and bottom (example below). The gray 10 background of the image is currently bleeding into the gray 10 background of the website page.
- There is also an extra space between the words "called" and "categories" in the description.

<img width="672" alt="Screenshot 2024-08-28 at 5 07 38 PM" src="https://github.com/user-attachments/assets/1084316a-48a2-4d6d-88c4-a56ba811a4f9">